go/vt/topotools/routing_rules.go

```diff
@@ -122,3 +122,40 @@ func SaveShardRoutingRules(ctx context.Context, ts *topo.Server, srr map[string]
 
 	return ts.SaveShardRoutingRules(ctx, srs)
 }
+
+// endregion
+
+// region keyspace routing rules
+
+func GetKeyspaceRoutingRulesMap(rules *vschemapb.KeyspaceRoutingRules) map[string]string {
```

Comment:
Can this function be private? It's used only in this file.

Reply:
Also used in `vindexes\vschema.go`